src/test/twirl/uk/gov/hmrc/play/views/layouts/test_includes_that_use_config.scala.html

```diff
@@ -0,0 +1,30 @@
+@import uk.gov.hmrc.play.config.OptimizelyConfig
+@import uk.gov.hmrc.play.config.AssetsConfig
+@()(implicit assetsConfig: AssetsConfig, optimizelyConfig: OptimizelyConfig)
+
+@*
+The HTML produced by this template doesn't need to make sense, it just
+needs to include the templates which use configuration to ensure that they
+can be rendered in tests without a running Play application (by passing in
+test AssetsConfig and OptimizelyConfig instances that do not access Play
+configuration).
+*@
```

Comment:
This could be made more concise:

```The HTML produced by this template just needs to include the templates which use configuration to ensure that they can be rendered in tests without a running Play application```